pom.xml

```diff
@@ -127,6 +127,11 @@
                 <artifactId>commons-lang3</artifactId>
                 <version>3.11</version>
             </dependency>
+          <dependency>
+            <groupId>org.jenkins-ci.main</groupId>
+            <artifactId>jenkins-test-harness</artifactId>
+            <version>1706.v5257fc59612a_</version>
```

Comment:
For cases where the Jenkins test harness needs to be overridden to a newer version, I think it is better to set the override the `jenkins-test-harness.version` property than to add an explicit version in a dependency or dependency management entry. The reason for this is that the latter makes it impossible to use a custom version when running with `mvn -Djenkins-test-harness.version` (since the property won't be consulted at all when an explicit version is used), while the former still keeps the property system in place but simply changes the default value.

Reply:
addressed in 03d36c31aec34cc94dbedac28cf8c7cccffdcc25